sources/configuration.ts

```diff
@@ -209,6 +220,19 @@ export class CatalogConfigurationReader {
     this.configCache.delete(workspaceRoot);
   }
 
+  /**
+   * Check if a workspace is ignored based on the configuration
+   */
+  async shouldIgnoreWorkspace(workspace: Workspace): Promise<boolean> {
+    const config = await this.readConfiguration(workspace.project);
+
+    if (config.options?.ignoredWorkspaces) {
+      return config.options.ignoredWorkspaces.includes(structUtils.stringifyIdent(workspace.manifest.name));
```

Comment:
It would be useful to be able to config glob patterns for ignored workspace

Reply:
You mean supporting glob patterns for workspace name, right? I'm considering using [picomatch](https://github.com/micromatch/picomatch) for this, what do you think?